config/default.js

```diff
@@ -55,7 +56,7 @@ module.exports = {
     domain: 'mozilla-iot.org',
     pagekite_cmd: './pagekite.py',
     port: 443,
-    certemail: 'certificate@mozilla-iot.org'
+    certemail: 'certificate@mozilla-iot.org',
   },
-  bcryptRounds: 2
+  bcryptRounds: 2,
```

Comment:
What's with all the extra commas?

Reply:
It's not JSON, so the trailing commas make maintenance easier.